src/partials/contacts.html

```diff
@@ -0,0 +1,69 @@
+<section class="contacts">
+    <div class="contacts__container">
+        <h3 class="contacts__display--none">CONTACTS</h3>
```

Comment:
Czemu tu dałeś ten nagłówek? Na figmie na żadnej wersji czy to desktop czy tablet nie ma tego nagłówka

Reply:
Tutaj jest nagłówek, ponieważ jest taka wytyczna dla tej sekcji, aby dodać niewidoczny nagłówek